clang/test/CheckedCRewriter/3d-allocationBUG.c

```diff
@@ -1,4 +1,6 @@
-// RUN: cconv-standalone %s -- | FileCheck -match-full-lines %s
+// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
```

Comment:
Let's use the above approach (both CHECK and CHECK_ALL / NO_ALL) for the PR #181 too. (I mentioned this in a comment there.)